src/core/Form/Textarea/Textarea.tsx

```diff
@@ -34,11 +38,35 @@ const textareaClassNames = {
   disabled: `${baseClassName}--disabled`,
   error: `${baseClassName}--error`,
   statusTextHasContent: `${baseClassName}_statusText--has-content`,
+  bottomWrapper: `${baseClassName}_bottomWrapper`,
+  characterCounter: `${baseClassName}_characterCounter`,
+  characterCounterError: `${baseClassName}_characterCounter--error`,
```

Comment:
${baseClassName}_characterCounter--error didn't seem to be in baseStyles if textarea

Reply:
I'll add the correct styling here as well.